### PR TITLE
Touchup README.md and fix whitespace issues

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

README.md had a lingering whitespace issues.  This will (hopefully) fix that and touches up a few other nits.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by baude** [comment]
bot, retest this please

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GO ?= go
-EPOCH_TEST_COMMIT ?= dd0d35d
+EPOCH_TEST_COMMIT ?= 05317d8
```
Comment (mheon) [inline Makefile]: It looks like there's trailing whitespace on this. Could that make gitvalidation angry?

Comment (baude) [reply Makefile]: did it pass gitvalidation locally ?

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
Just like #230, Error on Red Hat CI without any logs/errors.   Looks like the bot doesn't like me, can someone kick it please?

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
Still seeing gitvalidation errors - whitespace errors in 0aa2459b2de53bd3b46a1b34aaebc89116fdb040

**Comment by TomSweeneyRedHat** [comment]
Finally all green test buttons.  @mheon @baude @rhatdan sorry about the headaches, thanks for the help in fixing, and PTAL!  This should fix up the test errors plaguing others.

**Comment by mheon** [comment]
Going to merge to clean up gitvalidation issues

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**